cypress/e2e/pages/onDemandTV/tests.js

```diff
@@ -29,34 +26,15 @@ export default ({ service, pageType, variant }) => {
               );
             }
 
-            const language = appConfig[service][variant].lang;
-            const embedUrl = getEmbedUrl({ body: jsonData, language });
-            const isBrandPage = isBrand(jsonData);
-
-            cy.get('iframe').then(iframe => {
-              let iframeURL = isBrandPage ? iframe.prop('src') : embedUrl;
-              iframeURL = iframeURL.split('.com').pop();
-              cy.log(`cy.get('iframe') assertion has already happened`);
-              cy.log(
-                `used for Brand - iframe.prop('src') = ${iframe.prop('src')}`,
-              );
-              cy.log(`used for Episode - embedURL = ${embedUrl}`);
-              cy.log(`selector for iframe = iframe[src*="${iframeURL}"]`);
-              const pathTested = embedUrl.replace(
-                /^\//,
-                `${envConfig.baseUrl}/`,
-              );
-              cy.log(`path that will have response tested is ${pathTested}`);
-
-              cy.get(`iframe[src*="${iframeURL}"]`).should('be.visible');
-              cy.testResponseCodeAndTypeRetry({
-                // embedUrl may be relative - making it absolute to test the response
-                path: embedUrl.replace(/^\//, `${envConfig.baseUrl}/`),
-                responseCode: 200,
-                type: 'text/html',
-                allowFallback: true,
+            cy.get('[data-e2e="media-loader__container"]').should('be.visible');
+            cy.get('[data-e2e="media-loader__placeholder"] div')
+              .eq(1)
+              .within(() => {
+                cy.get('img')
+                  .should('be.visible')
+                  .should('have.attr', 'src')
+                  .should('not.be.empty');
```

Comment:
Should we check that the img source matches what's in the data, or is that overkill?

Reply:
This assertion has been added [here](https://github.com/bbc/simorgh/pull/11928/files#diff-5babdcd98a0017973821535dce88c8916435f66e93ae8287287dadf218f0f8e2R33) enabled by [videoPlaceholderImageUrl](https://github.com/bbc/simorgh/pull/11928/files#diff-be0dc0c9e10b2f5281a37e5c0dfc2a0999110ecd8e3fc0c7b4d0125611108e5aR60).